backend/app/lm/handlers.py

```diff
@@ -102,33 +103,29 @@ async def process_request(self) -> Resp:
         config_event = log_lm_config(ses, usr, arena_request_event, config)
         # Build the request
         lm_request = await self.lm_request().evaluate(session=self.session)
+        print('messagges_prompt',lm_request.content.messages[0])
+        print('messagges_doc',lm_request.content.messages[1])
         lm_request_event = arena_request_event
+        mapping_list = []   
```

Comment:
I think you want a unique mapping that you update with relevant items coming from messages of interest